aya/src/bpf.rs

```diff
@@ -375,6 +375,10 @@ impl<'a> EbpfLoader<'a> {
 
     /// Loads eBPF bytecode from a buffer.
     ///
+    /// The program will need to be aligned properly for you to use it.
```

Comment:
nit: I would say The buffer needs to be 4-bytes aligned. If you are ...

Reply:
done

Reply:
I still see the past tense being used.

Alessandro proposed:

```
The buffer needs to be 4-bytes aligned. If you are ...
```

In the diff, I see:

```
The buffer will need to be aligned to 4 bytes. If you are ...
```